sys-apps/tuned: Add 2.13.0 and use Python 3

ekohl opened the pull request:
Changes to tuned:
* Move from Python 2 to Python 3.
* Correct DOCDIR patching
* Optimize Python files
* Update XDG icon cache

This also includes dev-python/python-linux-procfs version 0.6.1 which adds Python 3 support and is a dependency. I'm not sure about the convention in this repository whether it's ok to combine both in a single PR.

Zlogene commented on sys-apps/tuned/tuned-2.13.0.ebuild:
```diff
@@ -0,0 +1,63 @@
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
```
keyword also for `~x86`

ekohl replied:
I've copied tuned-2.10.0 which already dropped it. ced5658b20ceb2b385067f4dc3c5546eb50929c3 states:
> 2) drop x86 keyword due to dev-python/python-linux-procfs

Zlogene commented on sys-apps/tuned/tuned-2.13.0.ebuild:
```diff
@@ -0,0 +1,63 @@
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
```
rename to `COMMON_DEPEND`

ekohl replied:
Also due to copying. Fixing.

a17r replied:
however, `DEPEND` contains none else than `COMMON_DEPEND` so this var is just superfluous. Just make it `DEPEND` here already and inherit that in `RDEPEND`.

Zlogene commented on sys-apps/tuned/tuned-2.13.0.ebuild:
```diff
@@ -0,0 +1,63 @@
+	sys-apps/dbus
+	sys-apps/ethtool
+	sys-power/powertop
+	sys-process/procps
```
you do not need this dependency

ekohl replied:
Also due to copying. How do you determine this isn't needed? Reading the spec file for dependencies?

Zlogene replied:
it is a part of the system set

Zlogene commented on sys-apps/tuned/tuned-2.13.0.ebuild:
```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
why not 7?

ekohl replied:
Because I copied the file and didn't think about it.

Zlogene commented on sys-apps/tuned/tuned-2.13.0.ebuild:
```diff
@@ -0,0 +1,63 @@
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1 systemd xdg-utils
```
`systemd` eclass is not used in the ebuild

Zlogene commented on sys-apps/tuned/tuned-2.13.0.ebuild:
```diff
@@ -0,0 +1,63 @@
+	default
+
+	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	python_fix_shebang "${ED}"
```
`D` should be enough here, prefix is not used anywhere in the ebuild.

ekohl commented:
I've pushed a separate commit that addresses the feedback for easy review. If this addresses the feedback I'll squash and force push with an updated message.
